airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/commands.py

```diff
@@ -3,26 +3,117 @@
 #
 
 """
-Module exposing the format command.
+Module exposing the format commands.
 """
+from __future__ import annotations
+
+import logging
+import sys
 
 import asyncclick as click
+from pipelines.airbyte_ci.format.configuration import FORMATTERS_CONFIGURATIONS, Formatter
+from pipelines.airbyte_ci.format.format_command import FormatCommand
 from pipelines.cli.click_decorators import click_ignore_unused_kwargs, click_merge_args_into_context_obj
-from pipelines.cli.lazy_group import LazyGroup
+from pipelines.helpers.cli import LogOptions, invoke_commands_concurrently, invoke_commands_sequentially, log_command_results
 from pipelines.models.contexts.click_pipeline_context import ClickPipelineContext, pass_pipeline_context
+from pipelines.models.steps import StepStatus
 
 
 @click.group(
-    cls=LazyGroup,
     name="format",
     help="Commands related to formatting.",
-    lazy_subcommands={
-        "check": "pipelines.airbyte_ci.format.check.commands.check",
-        "fix": "pipelines.airbyte_ci.format.fix.commands.fix",
-    },
 )
+@click.option("--quiet", "-q", help="Hide details of the formatter execution.", default=False, is_flag=True)
 @click_merge_args_into_context_obj
 @pass_pipeline_context
 @click_ignore_unused_kwargs
 async def format_code(pipeline_context: ClickPipelineContext):
     pass
+
+
+@format_code.group(
+    help="Run code format checks and fail if any checks fail.",
+    chain=True,
+)
+async def check():
+    pass
+
+
+@format_code.group(
+    help="Run code format checks and fix any failures.",
+    chain=True,
+)
+async def fix():
+    pass
+
+
+# Check and fix commands only differ in the export_formatted_code parameter value: check does not export, fix does.
+FORMATTERS_CHECK_COMMANDS = {
+    formatter: FormatCommand(formatter, export_formatted_code=False, **config) for formatter, config in FORMATTERS_CONFIGURATIONS.items()
+}
+
+FORMATTERS_FIX_COMMANDS = {
+    formatter: FormatCommand(formatter, export_formatted_code=True, **config) for formatter, config in FORMATTERS_CONFIGURATIONS.items()
+}
```

Comment:
Per "language" formatting commands are not hardcoded as function anymore. We dynamically create them from the `FORMATTERS_CONFIGURATIONS`. 
It avoids code duplication and allows a centralization of the configuration to simple and light module.